packages/bitcore-node/test/integration/models/block.integration.ts

```diff
@@ -489,7 +498,7 @@ describe('Block Model', function() {
       expect(unspentCoins[0].chain).to.equal('BTC');
       expect(unspentCoins[0].network).to.equal('regtest');
       expect(unspentCoins[0].mintTxid).to.equal('a2262b524615b6d2f409784ceff898fd46bdde6a584269788c41f26ac4b4919g'),
-      expect(unspentCoins[0].mintIndex).to.equal(0);
+        expect(unspentCoins[0].mintIndex).to.equal(0);
```

Comment:
space

Reply:
Ah! There's a comma instead of a semicolon on the line above. (I was wondering why Prettier would have added that space 😄)